src/pages/components/tabs/style.mdx

```diff
@@ -82,6 +100,67 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
   tabs.
 </Caption>
 
+### Dismissible line tabs
+
+| Type       | Element | Property         | Color token           |
+| ---------- | ------- | ---------------- | --------------------- |
+| Unselected | Tab     | background-color | transparent           |
+|            |         | border-bottom    | `$border-subtle` \*   |
+|            | Label   | text-color       | `$text-secondary`     |
+|            | Icon    | svg              | `$icon-secondary`     |
+| Selected   | Label   | text-color       | `$text-primary`       |
+|            | Icon    | svg              | `$icon-primary`       |
+|            | Tab     | border-bottom    | `$border-interactive` |
+
+<div className="image--fixed">
+
+![Example of selected and unselected dismissible line tabs](images/tab-style-9.png)
+
+</div>
+
+<Caption>
+Example of selected and unselected dismissible line tabs.
+</Caption>
+
+<br />
+
+#### Interactive state color
+
+<br />
+
+| State    | Element         | Property         | Color token        |
+| -------- | --------------- | ---------------- | ------------------ |
+| Hover    | Label           | text-color       | `$text-primary`    |
+|          | Icon            | svg              | `$icon-primary`    |
+|          |                 | background-color | `$background-hover` |
+|          | Tab             | border-bottom    | `$border-strong`   |
+| Focus    | Tab: unselected | border           | `$focus`           |
+|          | Tab: selected   | border           | `$focus`           |
+| Disabled | Label           | text-color       | `$text-disabled`   |
+|          | Icon            | svg              | `$icon-disabled`   |
+|          | Tab             | background-color | transparent        |
+|          |                 | border-bottom    | `$border-disabled` |
+
+<div className="image--fixed">
+
+![Examples of unselected close hover, unselected hover, unselected focus, and disabled states for dismissible line tabs](images/tab-style-11a.png)
+
+</div>
+
+<Caption>
+  Examples of unselected close hover, unselected hover, unselected focus, and disabled states for
+  dismissible line tabs.
+</Caption>
+
```

Comment:
```suggestion
<div className="image--fixed">

```

Reply:
I think this is also causing an issue, there is a closing `</div>` without an opening `div`